hibernate-core/src/test/java/org/hibernate/jpa/test/criteria/QueryBuilderTest.java

```diff
@@ -232,6 +235,8 @@ public void testMultiselectWithPredicates() {
 	}
 
 	@Test
+	@SkipForDialect(CockroachDB192Dialect.class)
+	@RequiresDialect(CockroachDB201Dialect.class)
```

Comment:
do we still need `@SkipForDialect` if we have provided `@RequiresDialect` already?

Reply:
Good point. Actually, I'm not sure how I should implement this. Previously, this test would run against all dialects. Now I want it to run against all dialects except for `CockroachDB192Dialect`. But if I just use `@SkipForDialect(CockroachDB192Dialect.class)` then this will also skip the test for `CockroachDB201Dialect` since it is a sub-class of the 19.2 dialect.

Reply:
have you tried with the property `strictMatching` it is present in both `@RequiresDialect` and `@SkipForDialect`?

Reply:
ah found it -- there is a `strictMatching` property i can use here

Reply:
@dreab8 thanks! I didn't refresh so I missed your comment.